client/my-sites/site-settings/wrap-settings-form.jsx

```diff
@@ -56,13 +66,11 @@ const wrapSettingsForm = getFormSettings => SettingsForm => {
 					nextProps.clearDirtyFields();
 					nextProps.markSaved();
 				} else {
-					let text;
-					switch ( nextProps.saveRequestError.error ) {
+					let text = nextProps.translate( 'There was a problem saving your changes. Please try again.' );
+					switch ( nextProps.siteSettingsSaveError ) {
 						case 'invalid_ip':
 							text = nextProps.translate( 'One of your IP Addresses was invalid. Please try again.' );
 							break;
-						default:
```

Comment:
>we want a general default message for both Jetpack Settings and SIte Settings, not just for SIte Settings

But even if `siteSettingsSaveError` applies only to Site Settings, the `default` case would still be triggered.

>Also, in upcoming PRs we'll probably update this to contain more cases, and maybe different default messages for Jetpack Settings and Site Settings.

This makes more sense, but maybe at that point we should move the default to be a fallback after we've tried to find an appropriate message. Like... `if ( ! text ) text = translate( '...' );`